src/site/content/en/fast/codelab-replace-gifs-with-video/index.md

```diff
@@ -155,11 +155,32 @@ Update the `<video>` with `<source>` elements that link to your cat-herd videos:
 
 {% Aside %}
 Browsers don't speculate about which `<source>` is optimal, so the order
-of `<source>`s matters. For example, if you specify an MP4 video first and the
+of `<source>` elements matters. For example, if you specify an MP4 video first and the
 browser supports WebM, browsers will skip the WebM `<source>` and use the MPEG-4
 instead. If you prefer a WebM `<source>` be used first, specify it first!
 {% endAside %}
 
+## Add captions and descriptions
+One last detail to take care of: accessibility.
+With GIFs, you can provide alternative text
+to describe the image to users with visual impairments.
+For videos, you need two elements:
+- A `<track kind="descriptions">` element that links to a [WebVTT](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API)
+  file describing the visual information in the video
+- A`<track kind="captions">` element that links to a WebVTT file
+  indicating that there isn't any audio
+
+For example:
+
+```html/3-4
+<video autoplay loop muted playsinline>
+  <source src="/images/cat-herd.webm" type="video/webm">
+  <source src="/images/cat-herd.mp4" type="video/mp4">
+  <track src="/images/captions_en.vtt" kind="captions" srclang="en">
+  <track src="/images/descriptions_en.vtt" kind="descriptions" srclang="en">
```

Comment:
nit: the instructions above describe the description file and then the caption file. But down here in the source we've reversed the order to go caption first, then description.